rpcs3/Emu/Cell/Modules/cellCamera.cpp

```diff
@@ -454,30 +390,24 @@ s32 cellCameraOpenEx(s32 dev_num, vm::ptr<CellCameraInfoEx> info)
 
 	const auto g_camera = fxm::get<camera_thread>();
 
-	if (!g_camera)
-	{
-		return CELL_CAMERA_ERROR_NOT_INIT;
-	}
-
-	s32 status = set_and_send_read_mode(dev_num, info->read_mode);
-	if (status != CELL_OK)
+	s32 status;
+	if ((status = cellCameraSetAttribute(dev_num, CELL_CAMERA_READMODE, info->read_mode, 0) != CELL_OK))
```

Comment:
move the ) to the left of the CELL_OK check
(just commenting to prevent accidental merge)